sys/netlink/netlink_snl.h

```diff
@@ -194,7 +206,8 @@ snl_allocz(struct snl_state *ss, int len)
 	if (data == NULL) {
 		uint32_t size = ss->lb->size * 2;
 
-		while (size < len + sizeof(struct linear_buffer))
+		while (size < CHERI_REPRESENTABLE_LENGTH(len) +
```

Comment:
I did not do a CRRL on the full allocation since size is always a power of 2, so it should always be a representable length.

Reply:
Could add a KASSERT? But probably not worth it.